python/nano/src/bigdl/nano/deps/ipex/ipex_inference_xpu_model.py

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from bigdl.nano.pytorch.model import AcceleratedLightningModule
+import torch
+import intel_extension_for_pytorch as ipex
+from bigdl.nano.utils.pytorch import apply_data_to_xpu
+from bigdl.nano.pytorch.context_manager import generate_context_manager
+
+
+class PytorchIPEXPUModel(AcceleratedLightningModule):
+
+    def __init__(self, model: torch.nn.Module, thread_num=None):
+        super().__init__(model)
+        self.model = model.to('xpu')
```

Comment:
Is this inplaced?

Reply:
inplace issue will be considerred in next PR.